cmd/utils/taiko_flags.go

```diff
@@ -15,6 +15,12 @@ var (
 		Name:  "taiko",
 		Usage: "Taiko network",
 	}
+	TaikoInternalShastaTimeFlag = cli.Uint64Flag{
+		Name:    "taiko.internal-shasta-time",
+		Usage:   "Override InternalShastaTime for Taiko internal network (seconds)",
+		Value:   10,
```

Comment:
```suggestion
		Value:   0,
```
Let's use `0` as the default value, since we have a flag now.

Reply:
fixed